src/fluree/http_api/handlers/ledger.clj

```diff
@@ -65,19 +65,24 @@
 (def create
   (error-catching-handler
     (fn [{:keys [fluree/conn content-type credential/did]
-          {{:keys [ledger txn defaultContext opts] :as body} :body} :parameters}]
-      (let [ledger-exists? (deref! (fluree/exists? conn ledger))]
+          {:keys [body]} :parameters}]
+      (log/debug "create body:" body)
+      (let [ledger         (get body "@id")
+            ledger-exists? (deref! (fluree/exists? conn ledger))]
         (log/debug "Ledger" ledger "exists?" ledger-exists?)
         (if ledger-exists?
           (let [err-message (str "Ledger " ledger " already exists")]
             (throw (ex-info err-message
                             {:response {:status 409
                                         :body   {:error err-message}}})))
-          (let [opts*    (cond-> (opts->context-type opts content-type)
-                           defaultContext (assoc :defaultContext defaultContext)
-                           did (assoc :did did))
+          (let [default-context (get body "@context")
```

Comment:
This isn't quite in line with what happens for `/transact`. For transactions, the top-level context is interpreted as the context for the transaction document itself, as well as for the data contained within the graph (which will be combined with any underlying default-context as applicable).

We use an iri for default context (defined [here](https://github.com/fluree/db/blob/main/src/fluree/db/constants.cljc#L36)).

Reply:
Thanks! I had misinterpreted that. Will fix.